From 769402f3eab85971d215ad2999c98592d7030a5f Mon Sep 17 00:00:00 2001
From: Markus Klinik <mkl@lambdanaut.net>
Date: Tue, 18 Jul 2023 09:25:04 +0200
Subject: [PATCH] build with protobuf 23.4-1

Arch gave me this version of protobuf in a recent system update
---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6eb00cf6..e6b192a9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,7 +117,9 @@ find_package ( Boost REQUIRED
   system
   )
 
+find_package (Protobuf CONFIG REQUIRED)
 find_package (Protobuf 3.0 REQUIRED)
+set (PROTOBUF_LIBRARIES protobuf::libprotobuf)
 set (PROTO_FILES
   src/modes/thread_view/webextension/messages.proto
   )
@@ -145,7 +147,7 @@ endif()
 # compile flags and options
 #
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++20")
 
 include_directories (
   ${GTKMM3_INCLUDE_DIRS}
